lib/watir-webdriver/locators/element/locator.rb

```diff
@@ -0,0 +1,249 @@
+module Watir
+  module Locators
+    class Element
+      class Locator
+        attr_reader :selector_builder
+        attr_reader :element_validator
+
+        WD_FINDERS = [
```

Comment:
Don't know. We can make it public, but let's keep that for followup refactoring.